sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollectionExpressionsSuite.scala

```diff
@@ -2522,4 +2522,24 @@ class CollectionExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper
           Date.valueOf("2017-02-12")))
     }
   }
+
+  test("SplitPart") {
```

Comment:
It seems can't directly test `SplitPart`. So just test `replacement`:
https://github.com/apache/spark/blob/b07ef74feb44d6f33538a47bdb54fc021b4a40d3/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala#L2864-L2866


```scala
checkEvaluation(SplitPart(Literal.create("11.12.13", StringType), Literal.create(".", StringType), Literal(3)), "13")
```

```
Caused by: org.apache.spark.SparkUnsupportedOperationException: [INTERNAL_ERROR] Cannot evaluate expression: split_part(11.12.13, ., 3)
	at org.apache.spark.sql.errors.QueryExecutionErrors$.cannotEvaluateExpressionError(QueryExecutionErrors.scala:71)
	at org.apache.spark.sql.catalyst.expressions.RuntimeReplaceable.eval(Expression.scala:369)
	at org.apache.spark.sql.catalyst.expressions.RuntimeReplaceable.eval$(Expression.scala:368)
	at org.apache.spark.sql.catalyst.expressions.SplitPart.eval(stringExpressions.scala:2859)
```

Reply:
Can we simply add more tests in the existing test case `test("elementAt")`? We just need to test the `outOfBoundValue`